lib/pf/ipset_cache.pm

```diff
@@ -0,0 +1,135 @@
+package pf::ipset_cache;
+
+=head1 NAME
+
+pf::ipset_cache -
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::ipset_cache
+
+=cut
+
+use strict;
+use warnings;
+use Moo;
+use CHI;
+use pf::log;
+use pf::util qw(pf_run);
+
+our $logger = get_logger();
```

Comment:
If the logger is declared here, does that mean we won't be able to see the subroutine name in the log lines ?